requirements/base.in

```diff
@@ -5,3 +5,4 @@ Django>=1.11,<2.0          # Web application framework
 django-model-utils         # Provides TimeStampedModel abstract base class
 django-celery
 django-crum
+django-rest-framework
```

Comment:
@matthugs this is the wrong package name for django-rest-framework, the correct one is called `djangorestframewor`: https://pypi.org/project/djangorestframework/#history

You'll see that the one you added here has only one release of 0.1.0 which is quite old.  This is causing edx-platform to try to install this package as well which is incorrect.

Can you please correct ASAP.

Reply:
addressed and merged to edx-plat master